### DISCUSSION TITLE
FragPipe generated SpecLib with mods

### DISCUSSION THREAD

**Discussion by veslugleynat** [vdemichev/DiaNN#1008] [Q&A]
Hi,

I have used GPF DIA data to generate a spectral library with mods(hydroxylation of proline and lysine) in FragPipe. I want to use this library to be able to search against mods in DIA-NN but it is creating empty output files. Can anyone who has done this before advise please?

Thanks in advance.

**Comment by vdemichev** [comment]
How do the logs look like?

**Reply by veslugleynat** [reply]
I have attached a screenshot of the last bit of the log. The run is finishing under 2 mins. pg_matrix, pr_matrix etc files are empty, main report has missing information under protein.names and genes. Thanks for the help!
![log](https://github.com/vdemichev/DiaNN/assets/142918506/fd431f32-827b-4d3d-8a12-3a9cad19c985)

**Reply by vdemichev** [reply]
Matrices are empty as nothing passed the 1% global FDR filter, which is expected with this kind of low numbers of peptides considered.
The Protein.Names and Genes columns are likely empty because protein inference was enabled while a FASTA database was not specified. So can just specify a FASTA database.
To improve the whole analysis, makes sense to also consider the background peptides, and this will in general boost the performance of DIA-NN.
